Framework/DataHandling/test/LoadInstrumentTest.h

```diff
@@ -293,10 +294,59 @@ class LoadInstrumentTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().remove(wsName);
   }
 
+  /// Test the Nexus geometry loader from file
+  void testExecNexusLOKI() {
```

Comment:
As far as I can tell, there is no new "destructive testing" happening with your changes. i.e deliberately trying to trigger known error situations. i.e. a nexus file that has no geometry ... etc. You should add some.